packages/social-previews/src/helpers.tsx

```diff
@@ -40,6 +40,15 @@ export const stripHtmlTags: Formatter< Array< string > > = ( description, allowe
 	return description ? description.replace( pattern, '' ) : '';
 };
 
+/**
+ * For social note posts we use the first 50 characters of the description.
+ * @param description The post description.
+ * @returns The first 50 characters of the description.
+ */
+export const getNoteTitle = ( description: string ): string => {
```

Comment:
This is pretty pedantic, but I wonder if this should be `getPreviewTitle` or `getTitleFromDescription` or something as there might be other title-less posts like the Aside post format.